### PR TITLE
Barebone /conncetors project as a dependency of /front

### PR DESCRIPTION
**Pull request by lasryaric**
Simply adding the /connectors project created via the temporal CLI create command and adding /connectors as a NPM dependency of /front.

### CODE REVIEW

File: connectors/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Hello World
```
Comment (spolu) [inline connectors/README.md:1]: Maybe clean that one up?

File: front/package-lock.json [hunk 2]
```diff
@@ -56,6 +57,31 @@
         "typescript": "^5.0.3"
       }
     },
+    "../connectors": {
+      "version": "0.1.2",
```
Comment (spolu) [inline front/package-lock.json:61]: When running `npm install`  from `front`:
```
spolu@spolu-dev ~/code/dust/front (aric/connectors-sub-project) $ git diff
diff --git a/front/package-lock.json b/front/package-lock.json
index 6252284..d45cab2 100644
--- a/front/package-lock.json
+++ b/front/package-lock.json
@@ -58,6 +58,7 @@
       }
     },
     "../connectors": {
+      "name": "dust-connectors",
       "version": "0.1.2",
       "dependencies": {
         "@slack/web-api": "^6.8.1",
spolu@spolu-dev ~/code/dust/front (aric/connectors-sub-project) $ 
```

Comment (fontanierh) [reply front/package-lock.json:61]: I guess it is due to the different node versions

**Review by fontanierh — DISMISSED** [review]
LGTM, happy to fix the nvm + eslint configs separately

File: front/package-lock.json (hunk 2 above)
Comment (lasryaric) [reply front/package-lock.json:62]: Solved 👍

**Review by fontanierh — APPROVED** [review]

**Pull request merged**